packages/url_launcher/url_launcher_android/pigeons/messages.dart

```diff
@@ -25,6 +27,16 @@ class WebViewOptions {
   final Map<String?, String?> headers;
 }
 
+/// Configuration options for in-app browser views.
+class BrowserOptions {
+  BrowserOptions({required this.showTitle});
+
+  /// Whether or not to show the webpage title.
+  ///
+  /// May not be supported on all platforms.
```

Comment:
This file is internal to the Android implementation, so this sentence doesn't make sense here.